packages/flame/test/components/priority_test.dart

```diff
@@ -1,27 +1,11 @@
+import 'dart:ui';
+
 import 'package:flame/components.dart';
 import 'package:flame_test/flame_test.dart';
 import 'package:test/test.dart';
 
-class _PriorityComponent extends Component {
-  _PriorityComponent(int priority) : super(priority: priority);
-}
-
-class _ParentWithReorderSpy extends Component {
-  int callCount = 0;
-
-  _ParentWithReorderSpy(int priority) : super(priority: priority);
-
-  @override
-  void reorderChildren() {
-    callCount++;
-    super.reorderChildren();
-  }
-
-  void assertCalled(int n) {
-    expect(callCount, n);
-    callCount = 0;
-  }
-}
+import '../custom_component.dart';
+// ignore_for_file: deprecated_member_use_from_same_package
```

Comment:
what does it use that is deprecated?

Reply:
I believe it's mostly calls like `game.children.changePriority(firstComponent, 11);`. Some time later we will refactor the test file to use more straightforward `firstComponent.priority = 11;`, but for now I wanted to leave everything mostly as-is to ensure that the change is non-breaking.